threedi_scenario_downloader/downloader.py

```diff
@@ -486,6 +507,20 @@ def download_total_damage_raster(
     )
 
 
+def download_arrival_time_raster(
+    scenario_uuid, projection, resolution, bbox=None, pathname=None
+):
+    """download Total Damage raster"""
+    download_raster(
+        scenario_uuid,
+        "depth-first-dtri",
+        projection,
+        resolution,
```

Comment:
Kijk, projection en resolution worden hier als positionele parameters meegegeven, wat gevaarlijk is als je de volgorde net hebt omgedraait. => heel erg oppassen.
Misschien het beste hier gewoon keyword arguments gebruiken? Dan kan het ook niet mis gaan.

Reply:
aangepast naar keyword arguments. Volgorde wijziging in changes.rst vermeldt.